api/src/pcapi/core/cookies/models.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import asdict
+from dataclasses import dataclass
+from datetime import datetime
+
+import sqlalchemy as sa
+from sqlalchemy import orm
+from sqlalchemy.ext import mutable
+from sqlalchemy.orm import Mapped
+
+from pcapi.models import Base
+from pcapi.models import Model
+from pcapi.models.pc_object import PcObject
+
+
+ListOfCookies = list[str]
+
+
+@dataclass
+class OptionalCookie:
+    accepted: ListOfCookies
+    refused: ListOfCookies
+
+
+@dataclass
+class Consent:
+    mandatory: ListOfCookies
+    optional: OptionalCookie
+
+
+class CookiesHistory(PcObject, Base, Model):  # type: ignore [valid-type, misc]
+    __tablename__ = "cookies_history"
+
+    id = sa.Column(sa.BigInteger, primary_key=True, autoincrement=True)
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏C'est redondant avec `PcObject` qui définit déjà un ID autoincrémenté